ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkboxgroup/v1/checkboxgroup/clientlibs/site/js/checkboxgroupview.js

```diff
@@ -108,21 +108,31 @@
             }, this)
         }
 
-        _updateEnabled(enabled) {
+        _updateEnabled(enabled,readOnly) {
             this.toggle(enabled, FormView.Constants.ARIA_DISABLED, true);
             this.element.setAttribute(FormView.Constants.DATA_ATTRIBUTE_ENABLED, enabled);
             let widgets = this.widget
             widgets.forEach(widget => {
-                if (enabled === false) {
+                if (readOnly === true) {
```

Comment:
if enabled is true and readOnly is true
this will set disabled = true on the element which is incorrect.
Also, L120 and L122 are conflicting with each other.

also, this function is changing aria-readonly state for all the elements even when the function is called ono change of enabled prop which is incorrect.

Reply:
this method must not modify the `readonly` prop of the component. 
it should be updated only in `_updateReadOnly` method